physics/Interstitials/UFS_SCM_NEPTUNE/GFS_photochemistry.F90

```diff
@@ -0,0 +1,105 @@
+! #########################################################################################
+!> \file GFS_photochemistry.f90
+!!
+! #########################################################################################
+module GFS_photochemistry
+  use machine,        only: kind_phys
+  use module_ozphys,  only: ty_ozphys
+  use module_h2ophys, only: ty_h2ophys
+  implicit none
+contains
+
+! #########################################################################################
+!> \section arg_table_GFS_photochemistry_init Argument Table
+!! \htmlinclude GFS_photochemistry_init.html
+!!
+! #########################################################################################
+  subroutine GFS_photochemistry_init(oz_phys_2006, oz_phys_2015, h2o_phys, errmsg, errflg)
+    logical, intent(in) :: &
+         oz_phys_2015, & ! Do ozone photochemistry? (2015)
+         oz_phys_2006, & ! Do ozone photochemistry? (2006)
+         h2o_phys        ! Do stratospheric h2o photochemistry?
+    character(len=*), intent(out) :: &
+         errmsg          ! CCPP Error message.
+    integer,  intent(out) :: &
+         errflg          ! CCPP Error flag.
+
+    ! Initialize CCPP error handling variables
+    errmsg = ''
+    errflg = 0
+
+    ! If no photchemical scheme is on, but SDF has this module, report an error?
+    ! DJS2024 Asks: If everything in _run() is controlled by these logicals, we probably don't
+    !         need to kill it here, since the run phase will be harmless?
+    if ((.not. oz_phys_2006) .and. (.not. oz_phys_2015) .and. (.not. h2o_phys)) then
```

Comment:
The logic in `_run` in principle allows you to run both oz_phys_2006 and oz_phys_2015. Maybe a good idea would be to check in the `_init` routine that not both are enabled.

Reply:
Good call. Will do.